packages/payment-detection/src/btc/address-based.ts

```diff
@@ -79,42 +80,49 @@ export default class PaymentNetworkBTCAddressBased {
     networkId: number,
   ): Promise<PaymentTypes.BTCBalanceWithEvents> {
     if (!request.extensions[paymentNetworkId]) {
-      throw new Error(`The request do not have the extension : ̀${paymentNetworkId}`);
+      return getBalanceErrorObject(
```

Comment:
I'm wondering if errors like this one shouldn't still throw, since it would only happen for an error in the code itself.
What do you think?